src/app/static/src/app/featureSwitches.ts

```diff
@@ -1,12 +1,10 @@
-const urlParams = new URLSearchParams(window.location.search);
+const urlParams = new URLSearchParams(globalThis.location?.search);
```

Comment:
playwright now loads this code because we import in the `DownloadType` into a playwright file, however it runs in a "node" environment not a browser env which means theres no `window` object. `globalThis` object is just the global this object so in a browser it is `window` and in a "node" env its just the `global` object, which means we can use this with playwright

Reply:
(Can you stick that in a comment above, I'll definitely forget about this)